content/english/jsappybird/1-Activity/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: "Activity 1"
```

Comment:
What does this file do?

Reply:
this is more like Index file which has all the items for Activity 1